spec/clusterable/ht_item_spec.rb

```diff
@@ -3,77 +3,98 @@
 require "spec_helper"
 require "clusterable/ht_item"
 require "cluster"
+require "hathifiles_database"
+
+RSpec.describe Clusterable::HtItem do
+  before(:all) do
+    hf_db = HathifilesDatabase.new(Services.holdings_db.uri)
+    hf_db.recreate_tables!
+  end
+
+  before(:each) do
+    Services.hathifiles_table.truncate
+  end
 
-RSpec.xdescribe Clusterable::HtItem do
   let(:ocn_rand) { rand(1_000_000).to_i }
   let(:item_id_rand) { rand(1_000_000).to_s }
   let(:ht_bib_key_rand) { rand(1_000_000).to_i }
   let(:htitem) { build(:ht_item) }
   let(:htitem_hash) { htitem.to_hash }
   let(:c) { create(:cluster, ocns: htitem_hash[:ocns]) }
 
-  before(:each) do
-    described_class.collection.find.delete_many
-    Cluster.collection.find.delete_many
+  def insert_htitem(htitem)
+    new_htitem_attrs = {
+      htid: htitem.item_id,
+      bib_num: htitem.ht_bib_key,
+      rights_code: htitem.rights,
+      access: htitem.access == "allow",
```

Comment:
we might want some clever & reusable way of mapping for access values (`allow/deny => 1/0`), or do the effort of making the switch to 1/0 to bring us closer to hf (if hf is as close to a gold standard as we'll ever have)

Reply:
Well -- ultimately I think we may want to not even use this and rather defer to the rights directly (recall we had discussed changing https://github.com/hathitrust/holdings-backend/blob/main/lib/reports/cost_report.rb#L151 to use the rights directly rather than "allow" / "deny"). If we do keep it in the long run, it would probably make more sense to have it on the model as a boolean rather than the string, and there could be some kind of presenter function for anything that needs to format/export it in a report. I think I'm inclined to keep this as-is for now and keep future opportunities in mind (https://github.com/hathitrust/holdings-backend/issues/317)